### ISSUE TITLE
Firefox, Error in NVDA, text navigation not possible when trying to copy text in google groups

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#9519]
### Steps to reproduce:
1. In Firefox, open a group in google groups (i.e. braillists)
https://groups.google.com/forum/#!forum/braillists/join
2. Press b to land on "new topic" and press enter
3. Make sure you have some text copied in your clipboard
4. In the message area, switch to focus mode and paste the text with ctrl+v

### Actual behavior:
The following error is appearing and text navigation is not possible. I must press tab and shift+tab to make it work again. This error appears also sometimes when typing or when adding new lines in the text field.

```
IO - inputCore.InputManager.executeGesture (22:11:55.145):
Input: kb(laptop):upArrow
IO - inputCore.InputManager.executeGesture (22:11:55.519):
Input: kb(laptop):upArrow
IO - inputCore.InputManager.executeGesture (22:11:55.835):
Input: kb(laptop):upArrow
DEBUG - editableText.EditableText._hasCaretMoved (22:11:56.239):
Caret didn't move before timeout. Elapsed: 100 ms
ERROR - scriptHandler.executeScript (22:11:56.283):
error executing script: <bound method Dynamic_EditableTextWithAutoSelectDetectionEditorMozillaIAccessible.script_caret_moveByLine of <NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionEditorMozillaIAccessible object at 0x052E3730>> with gesture u'Pfeiltaste nach oben'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 187, in executeScript
  File "editableText.pyc", line 185, in script_caret_moveByLine
  File "editableText.pyc", line 144, in _caretMovementScriptHelper
  File "editableText.pyc", line 130, in _caretScriptPostMovedHelper
  File "speech.pyc", line 781, in speakTextInfo
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 342, in getTextWithFields
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 305, in _getText
IndexError: pop from empty list
```

### Expected behavior:
No error should appear and text navigation should work like i.e. in Chrome or other browsers.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-17099,0d2a1525

#### Windows version:
Windows 10 1809 update

#### Name and version of other software in use when reproducing the issue:
Firefox 66.0.3

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

**Comment by Adriani90** [comment]
cc: @MarcoZehe, @jcsteh

**Comment by Adriani90** [comment]
This seems solved in NVDA 2020.1 with Firefox 76. Closing as works for me.
